src/components/search/SearchBar.jsx

```diff
@@ -1,8 +1,53 @@
-const SearchBar = () => {
+import React, { useState } from 'react';
+import searchIcon from '../../assets/icons/img_search.svg';
+import deleteSearchIcon from '../../assets/icons/delete-search.svg';
+
+const SearchBar = ({ onSearch }) => {
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const handleSearch = (event) => {
+    event.preventDefault();
+    onSearch(searchTerm);
+  };
+
+  const handleClear = () => {
+    setSearchTerm('');
+    onSearch('');
+  };
+
   return (
-    <>
-      <h1>Search Bar</h1>
-    </>
+    <div className="py-6 bg-lightGreen h-auto sm:h-24 flex justify-center items-center">
+      <form
+        className="border-2 bg-white border-white rounded-full h-12 w-3/4 flex justify-between ml-6 gap-1"
+        onSubmit={handleSearch}
+      >
+        <button type="submit" aria-label="Search">
+          <img src={searchIcon} className="h-6 mx-3" alt="search icon" />
+        </button>
+        <input
+          type="text"
+          className="focus:outline-none w-2/3"
+          aria-label="Search input"
+          placeholder="Search online classes or teachers..."
+          value={searchTerm}
+          onChange={(e) => setSearchTerm(e.target.value)}
+        />
+        <button
+          type="button"
+          onClick={handleClear}
+          aria-label="Clear search input"
+        >
+          <img src={deleteSearchIcon} className="" alt="clear search icon" />
+        </button>
+        <button
+          type="submit"
+          className="bg-darkGreen text-white font-roboto text-lg sm:text-xl md:text-2xl lg:text-3xl w-2/5 sm:w-1/2 md:w-1/3 lg:w-1/6 rounded-full"
+          aria-label="Search"
```

Comment:
here  the area-label is redundant: [By default, a button's accessible name is the content between the opening and closing [<button>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) tags, an image's accessible name is the content of its [alt](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#alt) attribute, and a form input's accessible name is the content of the associated [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) element.

If none of these options are available or if the default accessible name is not appropriate, use the aria-label attribute to define the accessible name of an element.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)

Reply:
Thank  you! Got it!